src/platform/site-wide/user-nav/components/SearchMenu.jsx

```diff
@@ -1,65 +1,204 @@
 import PropTypes from 'prop-types';
 import React from 'react';
+import { connect } from 'react-redux';
+import { toggleValues } from 'platform/site-wide/feature-toggles/selectors';
+import FEATURE_FLAG_NAMES from 'platform/utilities/feature-toggles/featureFlagNames';
 import classNames from 'classnames';
 import recordEvent from 'platform/monitoring/record-event';
+import debounce from 'platform/utilities/data/debounce';
 
 import { replaceWithStagingDomain } from '../../../utilities/environment/stagingDomains';
 import IconSearch from '@department-of-veterans-affairs/formation-react/IconSearch';
 import DropDownPanel from '@department-of-veterans-affairs/formation-react/DropDownPanel';
 
-import Typeahead, { isTypeaheadEnabled, typeaheadListId } from './Typeahead';
-
-class SearchMenu extends React.Component {
+export class SearchMenu extends React.Component {
   constructor(props) {
     super(props);
+    this.getSuggestions = debounce(
+      this.props.debounceRate,
+      this.getSuggestions,
+    );
     this.state = {
       searchAction: replaceWithStagingDomain('https://www.va.gov/search/'),
       userInput: '',
+      suggestions: [],
     };
   }
 
-  componentDidUpdate() {
-    this.refs.searchField.focus();
+  componentDidMount = () => {
```

Comment:
Just so we're aware, I think this is going to fire off this event every time a user goes to any page with search in the top nav, my guess is the analytics team won't find that meaningful but I could be wrong. Probably worth confirming with @jonwehausen from the Analytics team 🙂

Reply:
The ticket specifically called this tagging out, so I too will be interested to see if analytics agrees that they would like to keep it